actions.go

```diff
@@ -93,7 +93,7 @@ func AppCreateAction(clientConfig spinnaker.ClientConfig) cli.ActionFunc {
 			return errors.Wrapf(err, "submitting task")
 		}
 
-		resp, err := client.PollTaskStatus(ref.Ref, 1*time.Minute)
+		resp, err := client.PollTaskStatus(ref.Ref, time.Duration(cc.GlobalInt("timeout"))*time.Second)
```

Comment:
This doesn't seem right. ALl of the other calls are using `time.Minute`, but this is `time.Second`?

Reply:
I used time.Second because the timeout specified on the command line is in seconds.  Most utilities and configurations that I've seen timeouts in have been specified the timeout in seconds, not minutes.  That means I missed the time.Minutes changing to time.Seconds on the other two.  I'll fix it in a few minutes.